spinnaker_camera_driver/launch/camera.launch

```diff
@@ -60,6 +60,9 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
       <!-- Use the camera_calibration package to create this file -->
       <param name="camera_info_url" if="$(arg calibrated)"
              value="file://$(env HOME)/.ros/camera_info/$(arg camera_serial).yaml" />
+      <param name="histogram_equalization_enabled" value="true" />
```

Comment:
do you want the default value in the example launch file set to true?

Reply:
fixed